scripts/generators/foundry.py

```diff
@@ -1,20 +1,21 @@
-import yaml
 from glob import glob
 import json
 import os
-import pandas as pd
-import re
 from shutil import copy
 import fire
-import time
 import datetime
 
-secrets = json.load(open('../../secrets.json'))
+# Moves with fields that are problematic for the export
+# - Non-moves such as "Run Away" are excluded
```

Comment:
These are moves available to all Pokemon, may want to have them included. What's causing them not to work?

Reply:
Nothing, but these common actions already have their own fields on the character sheet. It'd be confusing if they're  included as moves as well, which is why I'm sticking to "official" moves that not every Pokémon has

Reply:
Aaaaah, makes sense. Not a bad idea either.